actions.jsx

```diff
@@ -189,7 +191,39 @@ export function setIndicator(indicator) {
 }
 
 
+export function fetchApplicationBootstrap() {
+  return dispatch => {
+    dispatch(requestApplicationBootstrap());
+    const bootstrapEndpoint = $.ajax({
+      type: 'GET',
+      url: 'https://nxt.staging.lizard.net/bootstrap/kpi/',
+      xhrFields: {
+        withCredentials: true,
+      },
+      success: (data) => {
+        return data;
+      },
+    });
```

Comment:
Nou, als ik dit doe:
```javascript
bootstrapEndpoint.then((data) => {
```
Dan klaagt verderop een aantal components over undefined variabelen. Kreeg het niet zo snel getraceerd wat de oorzaak is, en dit werkt prima zo.